tests/test_local_files.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+import os
+import skhep_testdata as skhtd
+
+
+data_dir = os.path.dirname(skhtd.__file__)
+data_dir = os.path.join(data_dir, "data")
+
+
+def test_data_path():
+    path = os.path.join(data_dir, "uproot-Zmumu.root")
+    assert skhtd.data_path("uproot-Zmumu.root") == path
+
+
+def test_data_path_missing():
+    path = skhtd.data_path("doesnt-exist.root", raise_missing=False)
+    assert path == os.path.join(data_dir, "doesnt-exist.root")
+
+    with pytest.raises(RuntimeError):
+        skhtd.data_path("doesnt-exist.root")
+
+    with pytest.raises(RuntimeError):
+        skhtd.data_path("doesnt-exist.root", raise_missing=True)
+
+
+def test_delegate_to_remote(monkeypatch, tmpdir):
+    def dummy_remote_file(filename, data_dir=None, raise_missing=False):
+        if not data_dir:
+            data_dir = str(tmpdir)
+        return os.path.join(data_dir, filename)
+
+    monkeypatch.setattr(skhtd.remote_files, "remote_file", dummy_remote_file)
```

Comment:
Apologies if I'm missing something trivial,but where is `monkeypatch` defined?

Reply:
Sorry, this isn't documented anywhere at this point, I'll add a mention to the README and once I set up the CI it should be more obvious.  But these are "fixtures" provided by default with `pytest`.
for now I'm running tests doing:
```bash 
pytest --cov=skhep_testdata -vv tests/
```
(for which you need the pytest-cov plugin installed).